docs/HowToMakeAProjectAnAresProject.md

```
@@ -106,18 +106,18 @@ Add the Ares library to both the agent configuration and test implementation, as
+    aresAgent "de.tum.cit.ase:ares:2.1.0:agent"
     aresAgent "org.aspectj:aspectjrt:1.9.25.1"
-    testImplementation "de.tum.cit.ase:ares:2.0.1-Beta9"
-    aspect "de.tum.cit.ase:ares:2.0.1-Beta9"
+    testImplementation "de.tum.cit.ase:ares:2.1.0"
+    aspect "de.tum.cit.ase:ares:2.1.0"
     implementation "org.aspectj:aspectjrt:1.9.25.1"
 }
 ```
 
 > **Tip (Gradle version catalog):** If your project uses a Gradle version catalog file (for example `gradle/<versions-catalog>.toml`), you can declare the version centrally:
 > ```toml
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)